packages/manager/modules/octavia-load-balancer/src/delete/delete.routing.js

```diff
@@ -0,0 +1,37 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from '../octavia-load-balancer.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('octavia-load-balancer.loadbalancers.delete', {
+    url: '/delete?loadBalancerId&loadBalancerName&loadBalancerRegion',
+    views: {
+      modal: {
+        component: 'octaviaLoadBalancerDelete',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      breadcrumb: () => null,
+      serviceName: /* @ngInject */ ($transition$) =>
+        $transition$.params().projectId,
+      loadBalancerId: /* @ngInject */ ($transition$) =>
+        $transition$.params().loadBalancerId,
+      loadBalancerRegion: /* @ngInject */ ($transition$) =>
+        $transition$.params().loadBalancerRegion,
+      loadBalancerName: /* @ngInject */ ($transition$) =>
+        $transition$.params().loadBalancerName,
+      goBack: /* @ngInject */ ($state) => () =>
+        $state.go('octavia-load-balancer'),
+      returnToListingSuccess: /* @ngInject */ ($state) => () =>
+        $state.go(
+          'octavia-load-balancer.loadbalancers',
+          {},
+          {
+            reload: true,
+          },
+        ),
```

Comment:
**nitpick:** No need for 2 callback functions for going to the same state. Refer `goBack` function in `packages/manager/modules/pci/src/projects/project/data-processing/notebooks/add-notebook/command/command.routing.js`

Reply:
This one is not the same because I need to reload loadbalancers' list after a successfull delete of one of them. That's the purpose of the "reload: true"